.github/workflows/multichain-e2e-template.yml

```diff
@@ -114,6 +119,11 @@ jobs:
         run: make register-bank-assets
         working-directory: ./agoric-sdk/multichain-testing
 
+      - name: Open Osmosis Liquidity Pool
```

Comment:
This step took ~12 seconds to finish, while the whole multichain-testing job took 44 minutes, I'm not sure if adding this flag is worth it

Reply:
If it makes more sense for the Liquidity Pool to be open and funded on Osmosis by default, I can remove the conditional check in the `Open Osmosis Liquidity Pool` step.
Additionally, in that case, I can update the `make start` target to automatically invoke the `open-liquidity-pool` target as part of its execution.